R/utilities-units.R

```diff
@@ -339,13 +335,23 @@ getDimensionByName <- function(name) {
 #' @examples
 #'
 #' ospsuite:::getUnitsEnum()
-#'
 #' @keywords internal
 getUnitsEnum <- function() {
   dimensions <- allAvailableDimensions()
 
   units <- lapply(dimensions, function(dimension) {
-    x <- getUnitsForDimension(dimension = dimension)
+    x <- tryCatch(
```

Comment:
@Yuri05 I cannot test this. Can you try to see if this workaround solves the problem.
I am also returning an "Unavailable" unit in this case so that the user sees that something is wrong

Reply:
The patch looks good to me.

Have you tried to see if you can reproduce the original problems using `Sys.setlocale()`?